app/frontend/src/features/profile/view/leaveReference/ReferenceForm.tsx

```diff
@@ -104,19 +107,19 @@ export default function ReferenceForm({ user }: ReferenceFormProps) {
       referenceData={referenceData}
       setReferenceValues={setReferenceValues}
     />
-  ) : step === steps[1] ? (
+  ) : step === `${RATING_STEP}` ? (
```

Comment:
```suggestion
  ) : step === RATING_STEP ? (
```
Unneeded template string